formats/hocon/build.gradle

```diff
@@ -14,15 +14,17 @@
  *  limitations under the License.
  */
 
-apply plugin: 'kotlin'
+apply plugin: 'kotlin-multiplatform'
```

Comment:
Will it affect publishing? I'd rather try to avoid making this subproject multiplatform

Reply:
Also, it would be nice to set up source dirs so actual files should not be moved

Reply:
It shouldn't negatively affect publishing

Reply:
Well, there's now `kotlinx-serialization-hocon` and `kotlinx-serialization-hocon-jvm` artifacts. The first one depends on the second, so users shouldn't face issues unless they're using maven: #1143